### PR TITLE
Feature/#607

### PR DESCRIPTION
**Pull request by HenkKodde**

### CODE REVIEW

**Comment by elnyry** [comment]
For the overview, can you please also use the diagram from the presentation on slide#15 here: https://github.com/mojaloop/docs/blob/master/WorkShops/Presentations/Mojaloop_Phase3_kick-off_Day1%2C2-Final.pdf

File: README.md [hunk 1]
```diff
@@ -1,76 +1,89 @@
 # Notification service
```
Comment (elnyry) [inline README.md]: Update the title to "Central Event Processor Service"

File: README.md [hunk 2]
```diff
@@ -1,76 +1,82 @@
...
+[![Git Commit](https://img.shields.io/github/last-commit/mojaloop/central-event-processor.svg?style=flat)](https://github.com/mojaloop/central-event-processor/commits/master)
+[![Git Releases](https://img.shields.io/github/release/mojaloop/central-event-processor.svg?style=flat)](https://github.com/mojaloop/central-event-processor/releases)
+[![Docker pulls](https://img.shields.io/docker/pulls/mojaloop/central-event-processor.svg?style=flat)](https://hub.docker.com/r/mojaloop/central-event-processor)
+[![CircleCI](https://circleci.com/gh/mojaloop/central-event-processor.svg?style=svg)](https://circleci.com/gh/mojaloop/central-event-processor)
+
+The Central Event Processor service  will provide notification(s) to participating FSPs and the Hub Operator for changes to Net Debit Cap or Position adjustments. Notification(s) are also provided for approaching the Net Debit Cap threshold that is configurable.
```
[the remark this answers is not in the document]
Comment (HenkKodde) [reply README.md]: Have replace with the suggested wording.

File: README.md [hunk 3]
```diff
@@ -1,76 +1,82 @@
...
+* [Scheduler flow](#12-scheduler-flow)
+* [Notifier flow (separate service)](#13-notifier-flow-separate-service)
+
+## 1. Deployment
+See the [onboarding guide](onboarding.md) for running the service locally.
+
+## 2. Stories
+### Central Event Processor
+
+### stories
```
[the remark this answers is not in the document]
Comment (HenkKodde) [reply README.md]: Have removed the Stories section and replaced it with Notes.

File: README.md (hunk 1 above)
Comment (HenkKodde) [reply README.md]: Title updated.

**Pull request merged**